x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.test.tsx

```diff
@@ -56,6 +56,6 @@ describe('Data Frame Analytics: <CreateAnalyticsForm />', () => {
     expect(options.at(2).props().value).toBe('regression');
 
     const row2 = euiFormRows.at(1);
-    expect(row2.find('p').text()).toBe('Enable advanced editor');
+    expect(row2.find('.euiSwitch').text()).toBe('Enable advanced editor');
```

Comment:
Can this target `EuiSwitch > *` or something similar, instead (maybe needs a data-test-subj somewhere?). Better to not target class names.